pkgs/top-level/all-packages.nix

```diff
@@ -9010,6 +9010,7 @@ in
   lilv-svn = callPackage ../development/libraries/audio/lilv/lilv-svn.nix { };
 
   lv2 = callPackage ../development/libraries/audio/lv2 { };
+  lv2-git = callPackage ../development/libraries/audio/lv2/lv2-git.nix { };
```

Comment:
I have tried 3 times now to rebase this to the lv2-git commit, but have failed... Sorry